Specialization/Vengeance.lua

```diff
@@ -117,6 +124,11 @@ function DemonHunter:VengeanceBrand()
 		return VG.SigilOfFlame;
 	end
 
+	-- infernal_stike,if=cooldown.fiery_brand.remains=0;
+	if cooldown[VG.FieryBrand].remains < 2 and cooldown[VG.InfernalStrike].ready then
```

Comment:
Actually I think this should be '= 0' and not '< 2', let me fix that.

Reply:
`== 0` is the same as `.ready`, or more proper `<= 0`